src/app/redux/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+import { combineReducers } from 'redux';
+import channelsState from '../../features/Chanels/redux/reducers/channelsReducer';
+import postsState from '../../features/Chanels/redux/reducers/postsReducer';
+
+export default combineReducers({ channelsState, postsState });
```

Comment:
make it simple, just
`{ channels, posts } `

However, if the channelsState or postsState keeps the page's state data, then it may be ok.